src/io/iter_prefetcher.h

```diff
@@ -131,7 +131,7 @@ class PrefetcherIter : public IIterator<DataBatch> {
     }
     return iter.Next(&out_);
   }
-  virtual const DataBatch &Value(void) const {
+  virtual const DataBatch& Value(void) const {
```

Comment:
I think both styles are fine: https://google.github.io/styleguide/cppguide.html#Pointer_and_Reference_Expressions

No need to change this

Reply:
I find the former difficult to read, but ok.